### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by adam-palazzo**
I went ahead and brought package.json up to date, with the most recent dependencies as well as adding a few other fields that weren't there before.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,26 @@
 {
   "name": "extractor",
   "version": "1.0.0",
-  "description": "",
+  "description": "code extracts id3 information from mp3 files for metadata use",
```
Comment (RichardLitt) [inline package.json]: Perhaps "Extract id3 tags from mp3 files"?

File: package.json [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
+  "homepage": "https://github.com/RichardLitt/extract-id3-tags-from-mp3s",
+  "bugs": "https://github.com/RichardLitt/extract-id3-tags-from-mp3s/issues",
+  "license": "MIT",
+  "keywords": [
+    "id3",
+    "mp3",
+    "floss",
```
Comment (RichardLitt) [inline package.json]: Remove `floss`. It's not relevant.

File: package.json [hunk 3]
```diff
@@ -1,15 +1,26 @@
 {
   "name": "extractor",
```
Comment (RichardLitt) [inline package.json]: Is this a good name? Can we name it something else?

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Almost good. Some changes.

**Comment by adam-palazzo** [comment]
Went ahead and made those changes Richard.

**Comment by RichardLitt** [comment]
Looks great. Thanks!

**Pull request merged**